code/modules/vehicles/mecha/mecha_actions.dm

```diff
@@ -14,6 +14,9 @@
 	chassis = null
 	return ..()
 
+/datum/action/vehicle/sealed/mecha/proc/set_chassis(passed_chassis)
```

Comment:
dmdoc this for statistical purposes

Reply:
https://github.com/tgstation/tgstation/pull/73779/commits/84c71eb11a50c4d15ac75627153e648b5e584130